Colors.xcassets/EsriBlue.colorset/Contents.json

```diff
@@ -0,0 +1,20 @@
+{
+  "colors" : [
+    {
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "0.763",
+          "green" : "0.476",
+          "red" : "0.000"
+        }
```

Comment:
The colors are picked from Esri Brand Guidelines https://webapps-cdn.esri.com/brand/guidelines.html . For Dark Mode counterparts, I made small adjustments based on the original color.